File: Q1 - Two Sum/solution.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
 var twoSum = function(nums, target) {
    // solution 1: 121ms, 42.4MB
    for (let index1=0; index1<nums.length; index1++){
        for (let index2=index1+1; index2<nums.length; index2++){
            if(nums[index1] + nums[index2] === target){
                return [index1,index2];
            };
        };
    };
    return [];
};

File: Q1 - Two Sum/solution.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        # solution 1: 3000ms, 15.6MB
        for index1 in range(0, len(nums)):
            for index2 in range(index1+1, len(nums)):
                if (nums[index1]+nums[index2] == target):
                    return [index1, index2]
